scripts/evalCrawler.R

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env Rscript
+
+library(stringr)
+library(readr)
+library(purrr)
+library(dplyr)
+library(pbapply)
+
+#Sys.setlocale('LC_ALL','C')
+regex <- list()
+
+regex[[1]]  <- c( "(?<!([\"a-zA-Z0-9_#]))(eval\\()" , "4" , "eval")
+regex[[2]]  <- c( "(?<!([\"a-zA-Z0-9_#]))(evalq\\()" , "5" , "evalq")
+regex[[3]]  <-  c("(?<!([\"a-zA-Z0-9_#]))(eval\\.parent\\()" , "11" , "eval.parent")
+regex[[4]]  <- c("(?<!([\"a-zA-Z0-9_#]))(local\\()" , "5" , "local")
+
+fileCrawlOne <- function(file, regex, pkgName) {
+  lines <- readLines(file)
+  lineCount <- length(lines)
+  lines <- gsub("(^(([\\s\\t]*)?#)).*", "\n", lines , perl = TRUE);
```

Comment:
formatting - in general check the formatting issues like superfluous spaces, etc.

Reply:
Acknowledged!